templates/repo/release/list.tmpl

```diff
@@ -76,7 +76,7 @@
 									{{$.locale.Tr "repo.released_this"}}
 								</span>
 								{{if .CreatedUnix}}
-									<span class="time">{{TimeSinceUnix .CreatedUnix $.locale}}</span>
+									<span class="time">{{TimeSinceUnix $.Context .CreatedUnix $.locale}}</span>
```

Comment:
Why does this not work when a user is unauthenticated but `nil` is fine?
```suggestion
									<span class="time">{{TimeSinceUnix nil .CreatedUnix $.locale}}</span>
```

Reply:
Shouldn't, in the worse case, `$.Context` be nil?

Reply:
The error:
```
Render failed, failed to render template: repo/release/list, error: template error: builtin(static):repo/release/list:80:30 : executing "repo/release/list" at <TimeSinceUnix $.Context .CreatedUnix $.locale>: error calling TimeSinceUnix: runtime error: invalid memory address or nil pointer dereference
----------------------------------------------------------------------
									<span class="time">{{TimeSinceUnix $.Context .CreatedUnix $.locale}}</span>
									                     ^
----------------------------------------------------------------------
```

Reply:
Haven't looked into details, just share some my experiences:

1. Golang template is not friendly to panic  https://github.com/golang/go/issues/59400 , you need to write some temp code to do "recover" to see what happens.
2. According to Golang standard, `ctx` should never be nil. The Golang standard library functions will panic if you feed nil to them.